### DISCUSSION TITLE
Error (Xcode): Framework not found Pods_Runner

### DISCUSSION THREAD

**Discussion by Arjun544** [codemagic-ci-cd/codemagic-docs#1900] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Hi, we are using Codemagic for running automated tests and building APK for Android & IPA for iOS, for 1 year. We never encountered the following error before.

All unit and integration tests are passing locally on my machine but our build is failing when we "Enable Flutter Driver tests". If we disable it, the build complete successfully.

Note: All unit tests are passing successfully if we disable Flutter Driver tests.

We have followed all the steps given in this discussion: https://github.com/orgs/codemagic-ci-cd/discussions/1462#discussioncomment-5289563

Passed Unit tests image: https://ibb.co/H7L0DXc

### Expected results

All unit and integration is supposed to pass and all Artifacts should be generated.

### Actual results

**flutter_driver logs**: https://www.transfernow.net/dl/20230722gRitTZi6

**Error**: 
Got dependencies!
Running pod install...                                             13.9s
Running Xcode build...                                          
Xcode build done.                                           67.4s
Failed to build iOS app
Error (Xcode): Framework not found Pods_Runner

Error (Xcode): Linker command failed with exit code 1 (use -v to see invocation)

Could not build the application for the simulator.
Application failed to start on attempt: 1
Application failed to start on attempt: 2
Application failed to start on attempt: 3
Application failed to start. Will not run test. Quitting.

**Comment by himesh-cm** [comment]
Hi @Arjun544, Let's continue the conversation on intercom.
Once we have a resolution, i can update the discussion here.

**Comment by himesh-cm** [comment, marked as answer]
Refer to the [GH ](https://github.com/flutter/flutter/issues/126768) issue.
